corehq/apps/data_cleaning/templates/data_cleaning/tables/table_with_controls.html

```diff
@@ -55,6 +55,16 @@
           {% blocktrans %}
             Records Selected
           {% endblocktrans %}
+          {% if table.num_visible_selected_records != table.num_selected_records %}
```

Comment:
Nit: `<` is probably more explicit. If, somehow, `num_visible_selected_records` was greater than `num_selected_records`, the warning wouldn't make sense.

Reply:
It's a nitpick. Regardless of the code behind this, the expression, as written, says that the message should be displayed whenever the number of records aren't equal. That could occur if the number of visible records is less than the number of selected records (which is the intent), or if the number of visible records is greater than the number of selected records, which I believe is intended to be impossible. Using `<` makes that more explicit, regardless of whether the code allows this to occur.